float32 init_heading
---
bool is_success